mvc_controller.hpp

```diff
@@ -463,6 +466,8 @@ class MvcController final
     MvcModel& model_;
     MvcView const& view_;
 
+    std::map<std::string, std::vector<wxString> > itemboxes_cache_;
```

Comment:
Very minor, but the space between `>`s is not needed any more in C++11
```suggestion
    std::map<std::string, std::vector<wxString>> itemboxes_cache_;
```